app/javascript/app/components/map/map-component.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import {
+  ComposableMap,
+  ZoomableGroup,
+  Geographies,
+  Geography
+} from 'react-simple-maps';
+import ReactTooltip from 'react-tooltip';
+import { southAfricaPaths } from 'utils/maps';
+
+import styles from './map-styles';
+
+class Map extends Component {
+  componentDidMount() {
+    setTimeout(
+      () => {
+        ReactTooltip.rebuild();
```

Comment:
what's the reason behind it?

Reply:
There seems to be an issue with the mouse listeners on the geometries https://github.com/zcreativelabs/react-simple-maps/issues/15#issuecomment-324920003 and they recommend that workaround in the example code.